Formula/m/mupdf.rb

```diff
@@ -40,12 +40,12 @@ class Mupdf < Formula
     depends_on "mesa"
   end
 
-  conflicts_with "mupdf-tools",
-    because: "mupdf and mupdf-tools install the same binaries"
+  conflicts_with "mupdf-tools", because: "both install the same binaries"
 
   def install
-    # Remove bundled libraries excluding `extract` and "strongly preferred" `lcms2mt` (lcms2 fork)
-    keep = %w[extract lcms2]
+    # Remove bundled libraries excluding `extract`, `gumbo-parser` (deprecated), and
+    # "strongly preferred" `lcms2mt` (lcms2 fork)
+    keep = %w[extract gumbo-parser lcms2]
```

Comment:
Can we file an issue upstream about their plans for `gumbo-parser`?

Reply:
https://bugs.ghostscript.com/show_bug.cgi?id=706736